<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MacBook-Pro-de-M.local" name="businessLogic.GroupAcademicDAOTest" tests="7" failures="1" timestamp="17 Jun 2021 09:02:35 GMT" time="1.07" errors="0">
  <testcase name="testDeletedLGCASucessful on testDeletedLGCASucessful(businessLogic.GroupAcademicDAOTest)" time="0.144" classname="businessLogic.GroupAcademicDAOTest"/>
  <testcase name="testGetlgacs on testGetlgacs(businessLogic.GroupAcademicDAOTest)" time="0.224" classname="businessLogic.GroupAcademicDAOTest"/>
  <testcase name="testAddLGAC on testAddLGAC(businessLogic.GroupAcademicDAOTest)" time="0.294" classname="businessLogic.GroupAcademicDAOTest">
    <failure type="log.BusinessException" message="DataBase connection failed ">
      <![CDATA[log.BusinessException: DataBase connection failed 
	at businessLogic.GroupAcademicDAO.addedLGACSucessful(GroupAcademicDAO.java:97)
	at businessLogic.GroupAcademicDAOTest.testAddLGAC(GroupAcademicDAOTest.java:62)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testAddLGAC on testAddLGAC(businessLogic.GroupAcademicDAOTest) -->
  <testcase name="testGetGroupAcademicById on testGetGroupAcademicById(businessLogic.GroupAcademicDAOTest)" time="0.066" classname="businessLogic.GroupAcademicDAOTest"/>
  <testcase name="testSave on testSave(businessLogic.GroupAcademicDAOTest)" time="0.117" classname="businessLogic.GroupAcademicDAOTest"/>
  <testcase name="testGetGroupAcademicByIdNotFound on testGetGroupAcademicByIdNotFound(businessLogic.GroupAcademicDAOTest)" time="0.062" classname="businessLogic.GroupAcademicDAOTest"/>
  <testcase name="testUpdate on testUpdate(businessLogic.GroupAcademicDAOTest)" time="0.14" classname="businessLogic.GroupAcademicDAOTest"/>
</testsuite> <!-- businessLogic.GroupAcademicDAOTest -->
